### PR TITLE
docs: added about signing gpg to old commit

### PR DESCRIPTION
**Pull request by ddevsr**
**Description**
Adding how to signing gpg to old commit.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Comment by kenjis** [comment]
Thank you.

By the way, is there anyone who uses `git commit -S` option in real life?
I've never used the option.

File: contributing/workflow.md [hunk 1]
```diff
@@ -179,6 +179,21 @@ For instance, to commit your work from a debugging session:
...
 
+### Signing GPG Old Commits
+
+Almost developer forgot signing GPG with params `-S` in commiting like `git commit -S -m "Signed GPG"`, you can signing old commit to keep secure.
+
+```console
+> git switch your-branch
+> git commit --amend --no-edit -n -S
+> git push --force-with-lease origin your-branch
+```
```
Comment (kenjis) [inline contributing/workflow.md:190]: This seems good, but what if we have more than one commit?

Comment (ddevsr) [reply contributing/workflow.md:191]: Updated if more than one commit

**Comment by ddevsr** [comment]
> Thank you.
> 
> By the way, is there anyone who uses `git commit -S` option in real life? I've never used the option.

@kenjis Yes, `-S` will be not used if `git config --global commit.gpgsign true` and `git config --global user.signingkey 3AC5C34371567BD2` already set. According to what I have given a note regarding this PR. I before never use too on my old repos, and recently applied in repo community based.

**Review by kenjis — APPROVED** [review]

**Pull request merged**